application/messages/claim.go

```diff
@@ -83,9 +83,20 @@ func ClaimPreapprovedQueueMessage(tx *pop.Connection, claim models.Claim) {
 	claimItem.LoadItem(tx, false)
 
 	data["estimate"] = "$-"
-	data["deductible"] = domain.PercentString(claim.GetDeductibleRate(tx))
 	data["repairThreshold"] = domain.Env.RepairThresholdString
 
+	deductibleRate := domain.PercentString(claim.GetDeductibleRate(tx))
+	claim.ClaimItems[0].LoadItem(tx, false)
+	claim.ClaimItems[0].Item.LoadCategory(tx, false)
+	if minimumDeductible := claim.ClaimItems[0].Item.Category.MinimumDeductible; minimumDeductible > 0 {
+		data["deductible"] = fmt.Sprintf(
+			"%s (subject to %s minimum)",
+			deductibleRate,
+			api.Currency(minimumDeductible))
+	} else {
+		data["deductible"] = deductibleRate
+	}
```

Comment:
Alternatively:

```suggestion
	claimItem.LoadItem(tx, false)
	claimItem.Item.LoadCategory(tx, false)

	data["estimate"] = "$-"
	data["repairThreshold"] = domain.Env.RepairThresholdString
	data["deductible"] = domain.PercentString(claim.GetDeductibleRate(tx))
	if claimItem.Item.Category.MinimumDeductible > 0 {
		data["deductible"] += fmt.Sprintf(" (subject to %s minimum)", api.Currency(claimItem.Item.Category.MinimumDeductible))
	}
```

Reply:
Can't do `+=` because `data["deductible"]` is `interface{}` not string. Error: "mismatched types interface{} and string"

Reply:
How about this then?

```suggestion
	claimItem.LoadItem(tx, false)
	claimItem.Item.LoadCategory(tx, false)

	deductible := domain.PercentString(claim.GetDeductibleRate(tx))
	if claimItem.Item.Category.MinimumDeductible > 0 {
		deductible += fmt.Sprintf(" (subject to %s minimum)", api.Currency(claimItem.Item.Category.MinimumDeductible))
	}

	data["estimate"] = "$-"
	data["deductible"] = deductible
	data["repairThreshold"] = domain.Env.RepairThresholdString
```